src/Tour.js

```diff
@@ -517,6 +517,7 @@ class Tour extends Component {
                         disabled={current === i || disableDotsNavigation}
                         showNumber={showNavigationNumber}
                         data-tour-elem="dot"
+                        className={current === i ? 'reactour__dot--active' : ''}
```

Comment:
I think it's better to position the contant `className` with the other ones [here](https://github.com/elrumordelaluz/reactour/blob/27b5dfc35767ead3732d883fe7cfb1ceb35a047d/src/Tour.js#L563-L573) using it something like:

```js
<Dot
  {...props}
  className={cn(CN.dot.base, {
    [CN.dot.active]: current === i,
  })}
/>
```